pom.xml

```diff
@@ -94,7 +94,7 @@
         <proto-plugin.version>0.6.1</proto-plugin.version>
 	<protobuf.version>3.19.6</protobuf.version>
 	<kotlin.version>1.7.10</kotlin.version>
-        <square-wireschema.version>4.3.0</square-wireschema.version>
+        <square-wireschema.version>4.4.3</square-wireschema.version>
```

Comment:
I remember that I couldn't go that high because of confluent back in the time, look like we had a update from confluent recently?

Reply:
yep, confluent 7.3.1 updated its own version to 4.4.3... which is compatible with 4.3.0 :p